src/main/java/net/datafaker/providers/base/Internet.java

```diff
@@ -176,26 +176,62 @@ public String httpMethod() {
         return resolve("internet.http_method");
     }
 
+    /**
+     * Consider usage of {@link net.datafaker.providers.base.Text#text} instead,
+     * look at net.datafaker.providers.base.TextTest#textShouldContain3RULowerCaseAnd5CustomSpecialSymbols()
```

Comment:
RULower? Regular upper lower?

Reply:
>textShouldContain3RULowerCaseAnd5CustomSpecialSymbols

means that text should contain 3 lower case letters from `RU` locale and 5 custom special symbols